README.md

```diff
@@ -1,9 +1,11 @@
 omxplayer(1) -- Raspberry Pi command line OMX player
 ====================================================
 
-OMXPlayer is a commandline OMX player for the Raspberry Pi. It was developed as
-a testbed for the XBMC Raspberry PI implementation and is quite handy to use
-standalone. 
+OMXPlayer is a command-line video player for the Raspberry Pi. This sends video
+data directly to the [OpenMAX](https://en.wikipedia.org/wiki/OpenMAX)
+accelerated graphics silicon and **does not require a desktop** or a window
+manager. It was developed as a testbed for the XBMC Raspberry PI implementation
```

Comment:
I'm not sure of this wording. There isn't actually any openmax related silicon. There is hardware video decoder on the GPU, and there is a software API that supports the OpenMAX API.

Reply:
Ok, thank you! I will try again.